webapp/components/team_general_tab.jsx

```diff
@@ -42,7 +42,7 @@ const holders = defineMessages({
     },
     openInviteTitle: {
         id: 'general_tab.openInviteTitle',
-        defaultMessage: 'Allow anyone to join this team'
+        defaultMessage: 'Allow any authenticated user to join this team'
```

Comment:
Using `authenticated` here sounds like you need to authenticate to the team. Maybe just drop the word? So `Allow any user to join this team`.